composer.json

```
@@ -10,22 +10,28 @@
   ],
   "require": {
     "php": "^8.1",
-    "symfony/http-kernel": "^6.2",
-    "illuminate/support": "^10.0",
-    "illuminate/database": "^10.0",
-    "illuminate/validation": "^10.0",
+    "symfony/http-kernel": "^6.2|^7.0",
+    "illuminate/support": "^10.0|^11.0",
+    "illuminate/database": "^10.0|^11.0",
+    "illuminate/validation": "^10.0|^11.0",
     "league/mime-type-detection": "^1.9"
   },
   "require-dev": {
-    "psy/psysh": "@stable",
+    "psy/psysh": "^0.11.22|^0.12",
     "mockery/mockery": "^1.3.1",
-    "phpunit/phpunit": "^9.0",
-    "laravel/framework": "^10.0",
-    "orchestra/testbench": "^8.0",
-    "orchestra/testbench-dusk": "^8.0",
+    "phpunit/phpunit": "^9.0|^10.0",
+    "laravel/framework": "^10.0|^11.0",
+    "orchestra/testbench": "^8.0|^9.0",
+    "orchestra/testbench-dusk": "^8.0|^9.0",
     "calebporzio/sushi": "^2.1",
-    "laravel/prompts": "^0.1.6"
+    "laravel/prompts": "^0.1.6|dev-main"
```

```suggestion
    "laravel/prompts": "^0.1.6"
```

Symfony 7 support is available via `0.1.12`